services/vehicle-assembly-process-defect-detection-model-service/app/routers/model.py

```
@@ -0,0 +1,196 @@
+
+    except Exception as e:
+        logger.error(f"모델 정보 조회 중 오류 발생 - Request ID: {request_id}, 오류: {str(e)}", exc_info=True)
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail={
+                "success": False,
+                "message": "모델 정보 조회 중 오류가 발생했습니다",
+                "request_id": request_id,
+                "timestamp": datetime.now().isoformat()
+            }
+        )
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)